src/test/java/seedu/address/logic/parser/EditAppointmentCommandParserTest.java

```diff
@@ -29,7 +30,7 @@ public class EditAppointmentCommandParserTest {
     private EditAppointmentCommandParser parser = new EditAppointmentCommandParser();
 
     @Test
-    public void parse_missingParts_failure() {
+    public void parse_missingParts_failure() throws CommandException {
```

Comment:
tests like this shouldnt be throwing the exceptions. If there is an exception being thrown means the tests got something wrong unless its intentional. If its intentional, should assert that command exception is being thrown